requirements.txt

```diff
@@ -6,7 +6,7 @@
 # This file is autogenerated by pip-compile with python 3.7
 # To update, run:
 #
-#    pip-compile --no-emit-index-url
```

Comment:
`--no-emit-index-url` should be still used I think

Reply:
done